get_nxpRdLib.sh

```diff
@@ -3,18 +3,6 @@
 blue='\033[0;34m'
 NC='\033[0m' # No Color
 
-prereq() {
-  PREREQS="build-essential cmake $1"
-  
-  if dpkg -s $PREREQS > /dev/null 2>&1; then
-    echo -e "Prerequisites already installed: $PREREQS"
-  else
-    echo -e "[${blue}Installing prerequisites${NC}]"
```

Comment:
I'm willing to give the issue template approach a shot.  I may reimplement this check if it proves to be too much trouble.  The vast majority of users are using Raspbian, so this check was implemented with that in mind.  I can see how it's not ideal for non-Raspbian users.